presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/properties/LogicalPropertiesImpl.java

```diff
@@ -388,7 +382,7 @@ LogicalPropertiesImpl build()
             requireNonNull(sourceProperties, "sourceProperties is null");
             requireNonNull(keyVariables, "keyVariables is null");
             requireNonNull(outputVariables, "outputVariables is null");
-            checkArgument(!keyVariables.isEmpty(), "keyVariables is empty");
+            Preconditions.checkArgument(!keyVariables.isEmpty(), "keyVariables is empty");
```

Comment:
`import static com.google.common.base.Preconditions.checkArgument;` and remove `Preconditions.` from `Preconditions.checkArgument`

Reply:
Resolved in latest commit.